mw-webapp/src/dataAccessLogic/getDayReports.ts

```diff
@@ -27,6 +26,7 @@ export const getDayReports = async (): Promise<DayReport[]> => {
     return jobDone;
   });
 
+
   const plansForNextPeriod = firstReport.plansForNextPeriod
```

Comment:
why 2 empty line?

Reply:
The absence a prettier who would rule such things for me, fix)